src/Surfnet/ServiceProviderDashboard/Application/ViewObject/EntityList.php

```diff
@@ -48,6 +49,21 @@ public function getIterator()
         return new ArrayIterator($this->entities);
     }
 
+    public function hasTestEntities() : bool
+    {
+        $hasTestEntities = false;
+        foreach ($this->getEntities() as $entity) {
+            if ($entity->getEnvironment() === Constants::ENVIRONMENT_TEST
+                && $entity->getState() === Constants::STATE_PUBLISHED
+            ) {
+                $hasTestEntities = true;
+                break;
```

Comment:
Can be reduced to a `return true`, no need to keep a `$hasTestEntities` variable. But this is fine too!

Reply:
good one! 🚀